### PR TITLE
Register z.is-a.dev

### PR DESCRIPTION
**Pull request by ZenZoyaUDR**
Register z.is-a.dev with CNAME record pointing to vortres.netlify.app.

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
I cannot reach your website.

**Comment by ZenZoyaUDR** [comment]
wired i can access the site

File: domains/z.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "ZenZoyaUDR",
+           "email": "",
+           "discord": "915228509440454707",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.k0NezLkzuwzU94IcTvdb8q79AcHOS5Zf1fRq3aLXEYPI12ui81wuYWEPsvC9mEW6Y4lxFoZ12mgjobbFP2KBd_k28qrinDSamMs4MQr2yxnANP8pmi9JVvoXNDt8mH8I7Xns6w2NSa-fG89qbf9H7TKzucs3YurLQ5351RPcIpxKhHYpiGw45CiNNHg0K1CE2024tQ0Dkv2pzuJJYPvI2rDaHBr9t8azTrw8GddH38d58LNZx3-J5zYJKYTVCZZKiyR4AXHWomNCtn_7K0H0wj1F1KSA_SYqG-GwIdh0AZrbHRIwedtuLyfhqyS-i_ZcH_4zOHQCSL4a-qZiUWfTrg.DCxJYLyx2D66Nw16flyrcg.FEqGwXQefGYZ7POOSvkXoEnHEMMYpaB4xEVP_JndpWmqHe3Z9jJUWDiq3fkWtSeIzn3wU_nSnqXq7_sqjNypmIa7k9DjBTJIAPoQlszlb2s.jIEPIZhhp8J9Qqb4NW2p4g"
+        },
+    
+        "record": {
+            "CNAME": "vortres.netlify.app"
```
Comment (CuteDog5695) [inline domains/z.json:10]: Site works for me, but could you add more stuff to the website?

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**